DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan/test_vlan_default_configuration.py

```diff
@@ -0,0 +1,259 @@
+import asyncio
+
+import pytest
+from dent_os_testbed.utils.test_utils.tgen_utils import (tgen_utils_get_dent_devices_with_tgen,
+                                                         tgen_utils_setup_streams, tgen_utils_start_traffic,
+                                                         tgen_utils_stop_traffic, tgen_utils_get_traffic_stats,
+                                                         tgen_utils_get_loss, tgen_utils_dev_groups_from_config,
+                                                         tgen_utils_stop_protocols,
+                                                         tgen_utils_traffic_generator_connect, )
+from dent_os_testbed.utils.test_utils.vlan_utils import (configure_bridge_setup, configure_vlan_setup,
+                                                         get_traffic_port_vlan_mapping)
+
+pytestmark = pytest.mark.suite_functional_vlan
+
+packet_vids = ['X', 1, 2]  # VLAN tag number contained in the transmitted packet(s)
+port_map = ({"port": 0, "settings": [{"vlan": 1, "untagged": True, "pvid": True}]},
+            {"port": 1, "settings": [{"vlan": 1, "untagged": True, "pvid": True}]},
+            {"port": 2, "settings": [{"vlan": 1, "untagged": True, "pvid": True}]},
+            {"port": 3, "settings": [{"vlan": 1, "untagged": True, "pvid": True}]},)
+
+
+@pytest.mark.asyncio
+@pytest.mark.usefixtures("cleanup_bridges")
+async def test_vlan_dafault_configuration_broadcast(testbed):
+    """
+    Test Name: Broadcast with default VLAN configuration
+    Test Suite: suite_functional_vlan
+    Test Overview: Test VLAN with broadcast on default configuration
+    Test Procedure:
+    1. Initiate test params.
+    2. Set links to vlans.
+    3. Map receiving and non receiving dut_ports.
+        Port 1 ->  tx_port
+        Ports 2, 3, 4 -> rx_ports
+    4. Setup packet stream(s) for the broadcast packet:
+    5. Send traffic to rx_ports
+    5.1 Verify traffic on rx ports
+    """
+
+    # 1. Initiate test params.
+    tgen_dev, dent_devices = await tgen_utils_get_dent_devices_with_tgen(testbed, [], 4)
+    if not tgen_dev or not dent_devices:
+        print("The testbed does not have enough dent with tgn connections")
+        return
+    device = dent_devices[0].host_name
+    tg_ports = tgen_dev.links_dict[device][0]
+    dut_ports = tgen_dev.links_dict[device][1]
+
+    await configure_bridge_setup(device, dut_ports)
+
+    # 2. Set links to vlans.
+    await configure_vlan_setup(device, port_map, dut_ports)
```

Comment:
Once you do VLAN configuration it is not a default one anymore.

Reply:
Your port_map corresponds to a default configuration, but you should not do any VLAN configuration steps. Just add ports to a VLAN-aware bridge.

Reply:
im adding `default_pvid=1` on 49 line, kind of the same?

Reply:
Your test is named `test_vlan_*default_configuration*_broadcast`. You don't need to call `configure_vlan_setup` here.

Reply:
I'm adding `VLAN 2` as well, that's why calling `configure_vlan_setup `

Reply:
Then I don't understand this test setup. That wouldn't make it `default_configuration` anymore.

Reply:
This is how MRVL did they step...